src/Illuminate/Foundation/Console/ClearCompiledCommand.php

```diff
@@ -27,15 +28,6 @@ class ClearCompiledCommand extends Command
      */
     public function fire()
     {
-        $compiledPath = $this->laravel->getCachedCompilePath();
-        $servicesPath = $this->laravel->getCachedServicesPath();
-
-        if (file_exists($compiledPath)) {
-            @unlink($compiledPath);
-        }
-
-        if (file_exists($servicesPath)) {
-            @unlink($servicesPath);
-        }
+        ComposerScripts::clearCompiled();
```

Comment:
Looked fine at first glance, but maybe this change is breaking too.

Instead of using the running Application instance, it creates a new one, and they could have different paths set.